plenum/server/req_handler.py

```diff
@@ -58,8 +67,18 @@ def commit(self, txnCount, stateRoot, txnRoot) -> List:
         self.state.commit(rootHash=stateRoot)
         return txnsWithSeqNo(seqNoStart, seqNoEnd, committedTxns)
 
-    def onBatchCreated(self, stateRoot):
+    def onBatchCreated(self, state_root):
         pass
 
     def onBatchRejected(self):
         pass
+
+    def is_query(self, txn_type):
+        return txn_type in self.query_types
+
+    def get_query_response(self, request):
```

Comment:
Please use abstractmethod annotation instead

Reply:
`abstractmethod` will force any extending class to implement this method but a request handler might not support querying